src/BenchmarksServer/Startup.cs

```diff
@@ -578,15 +599,37 @@ private static void DockerCleanUp(string containerId, string imageName)
 
             if (OperatingSystem == OperatingSystem.Windows)
             {
-                ProcessUtil.Run("cmd", "/c build.cmd /t:noop",
+                // Install latest stable 2.0 SDK version (and associated runtime)
+                ProcessUtil.Run("cmd", $"/c dotnet-install.ps1 --channel 2.0",
+                    workingDirectory: buildToolsPath,
+                    environmentVariables: env);
+
+                // Install latest SDK version (and associated runtime)
+                ProcessUtil.Run("cmd", $"/c dotnet-install.ps1 --version {sdkVersion}",
+                    workingDirectory: buildToolsPath,
+                    environmentVariables: env);
+
+                // Install runtime required by coherence universe
+                ProcessUtil.Run("cmd", $"/c dotnet-install.ps1 --version {latestRuntimeVersion} --shared-runtime",
                     workingDirectory: buildToolsPath,
                     environmentVariables: env);
             }
             else
             {
-                ProcessUtil.Run("/usr/bin/env", "bash build.sh /t:noop",
+                // Install latest stable 2.0 SDK version (and associated runtime)
+                ProcessUtil.Run("/usr/bin/env", $"bash dotnet-install.sh --channel 2.0",
+                    workingDirectory: buildToolsPath,
+                    environmentVariables: env);
+
+                // Install latest SDK version (and associated runtime)
+                ProcessUtil.Run("/usr/bin/env", $"bash dotnet-install.sh --version {sdkVersion}",
```

Comment:
Why do we need two sdks?

Reply:
Benchmarks can be run on the latest released SDK (2.0.3) or the one Universe is currently using (2.1.x). But writing that I assume your question is more about now that we know which one we use, we can only download one ... That would make sense, and this is not something we could do before with KoreBuild.

Reply:
Benchmark run on runtimes, not on SDK's we might need to have multiple runtimes installed but one SDK should be sufficient.